src/main/java/net/glowstone/util/Wearable.java

```diff
@@ -0,0 +1,120 @@
+    private static boolean intalized = false;
+    
+    private class WearableReturn {
+        private EnchantmentTarget wearablePosition;
+        private int armorPoints;
+        
+        public WearableReturn(EnchantmentTarget position, int points) {
+            this.wearablePosition = position;
+            this.armorPoints = points;
+        }
+        public EnchantmentTarget wearPosition() {
+            return wearablePosition;
+        }
+        public int armorPoints() {
+            return armorPoints;
+        }
+    }
+    
+    protected Map<Material, WearableReturn> createMap() {
+        Map<Material, WearableReturn> map2 = new HashMap<>();
+        map2.put(Material.LEATHER_BOOTS, new WearableReturn(EnchantmentTarget.ARMOR_FEET, 1));
+        map2.put(Material.LEATHER_LEGGINGS, new WearableReturn(EnchantmentTarget.ARMOR_LEGS, 2));
+        map2.put(Material.LEATHER_CHESTPLATE, new WearableReturn(EnchantmentTarget.ARMOR_TORSO, 3));
+        map2.put(Material.LEATHER_HELMET, new WearableReturn(EnchantmentTarget.ARMOR_HEAD, 1));
+        map2.put(Material.GOLD_BOOTS, new WearableReturn(EnchantmentTarget.ARMOR_FEET, 1));
+        map2.put(Material.GOLD_LEGGINGS, new WearableReturn(EnchantmentTarget.ARMOR_LEGS, 3));
+        map2.put(Material.GOLD_CHESTPLATE, new WearableReturn(EnchantmentTarget.ARMOR_TORSO, 5));
+        map2.put(Material.GOLD_HELMET, new WearableReturn(EnchantmentTarget.ARMOR_HEAD, 2));
+        map2.put(Material.IRON_BOOTS, new WearableReturn(EnchantmentTarget.ARMOR_FEET, 2));
+        map2.put(Material.IRON_LEGGINGS, new WearableReturn(EnchantmentTarget.ARMOR_LEGS, 5));
+        map2.put(Material.IRON_CHESTPLATE, new WearableReturn(EnchantmentTarget.ARMOR_TORSO, 6));
+        map2.put(Material.IRON_HELMET, new WearableReturn(EnchantmentTarget.ARMOR_HEAD, 2));
+        map2.put(Material.CHAINMAIL_BOOTS, new WearableReturn(EnchantmentTarget.ARMOR_FEET, 1));
+        map2.put(Material.CHAINMAIL_LEGGINGS, new WearableReturn(EnchantmentTarget.ARMOR_LEGS, 4));
+        map2.put(Material.CHAINMAIL_CHESTPLATE, new WearableReturn(EnchantmentTarget.ARMOR_TORSO, 5));
+        map2.put(Material.CHAINMAIL_HELMET, new WearableReturn(EnchantmentTarget.ARMOR_HEAD, 2));
+        map2.put(Material.DIAMOND_BOOTS, new WearableReturn(EnchantmentTarget.ARMOR_FEET, 3));
+        map2.put(Material.DIAMOND_LEGGINGS, new WearableReturn(EnchantmentTarget.ARMOR_LEGS, 6));
+        map2.put(Material.DIAMOND_CHESTPLATE, new WearableReturn(EnchantmentTarget.ARMOR_TORSO, 8));
+        map2.put(Material.DIAMOND_HELMET, new WearableReturn(EnchantmentTarget.ARMOR_HEAD, 3));
+        map2.put(Material.SKULL_ITEM, new WearableReturn(EnchantmentTarget.ARMOR_HEAD, 0));
+        map2.put(Material.ELYTRA, new WearableReturn(EnchantmentTarget.ARMOR_TORSO, 0));
+        map2.put(Material.PUMPKIN, new WearableReturn(EnchantmentTarget.ARMOR_HEAD, 0));
+        return map;
+    }
+    
+    public static void intalize() {
```

Comment:
Since I didn't realize I could set map as `new HashMap<>()` before, I was defining map2 as a local variable, but whenever I tried to access it NetBeans yelled at me for "accessing a non-static variable from a static method"